silkworm/rpc/http/request_handler.cpp

```diff
@@ -33,17 +33,16 @@ constexpr std::size_t kStreamBufferSize{4096};
 
 Task<void> RequestHandler::handle(const std::string& content) {
     auto start = clock_time::now();
-    Channel::Response msg_response;
+    std::string response{};
```

Comment:
No need to initialise empty string

Reply:
ok, I have understood that tidy_clang request it